lib/grpcApi/handlers/applyStateTransitionHandlerFactory.js

```diff
@@ -0,0 +1,86 @@
+const { ApplyStateTransitionResponse } = require('@dashevo/drive-grpc');
+const InvalidSTPacketError = require('@dashevo/dpp/lib/stPacket/errors/InvalidSTPacketError');
+
+const StateTransition = require('../../blockchain/StateTransition');
+const InvalidArgumentGrpcError = require('../error/InvalidArgumentGrpcError');
+const InternalGrpcError = require('../error/InternalGrpcError');
+
+/**
+ * @param {MongoDBTransaction} stateViewTransaction
+ * @param {DashPlatformProtocol} dpp
+ * @param {applyStateTransition} applyStateTransition
+ * @returns {applyStateTransitionHandler}
+ */
+module.exports = function applyStateTransitionHandlerFactory(
+  stateViewTransaction,
+  dpp,
+  applyStateTransition,
+) {
+  /**
+   * Apply received stPacket and stHeader to database inside transaction, opened earlier
+   *
+   * @typedef applyStateTransitionHandler
+   * @param {Object} call
+   * @returns {Promise<CommitTransactionResponse>}
+   */
+  async function applyStateTransitionHandler(call) {
+    const { request } = call;
+
+    const blockHeight = request.getBlockHeight();
+    const blockHashArray = Buffer.from(request.getBlockHash()).toString('hex');
+    const stPacketArray = Buffer.from(request.getStateTransitionPacket());
```

Comment:
`stHeaderBuffer`, `stPacketBuffer`

Reply:
It's binary array. 

Thre is an error here. It should look like this
`
const stPacketBinaryArray = request.getStateTransitionPacket();
if (!stPacketBinaryArray) {
      throw new InvalidArgumentGrpcError('stateTransitionPacket is not specified');
    }

const stPacket = Buffer.from(stPacketBinaryArray).toString('hex');
`